forch/forchestrator.py

```diff
@@ -247,7 +247,8 @@ def _initialize_orchestration(self):
             self._faucet_collector.set_device_state_reporter(self._device_report_server)
 
         self._port_state_manager = PortStateManager(
-            self._faucetizer, self, self._device_report_server, sequester_segment)
+            self._faucetizer, self, self._device_report_server,
+            sequester_segment=sequester_segment, sequester_timeout=sequester_timeout)
```

Comment:
is now the time to create a generic "config" bucket to pass in... or leave it as is? Not sure myself -- just asking the question. This is how constructors with 20 arguments start!